src/components/Grid.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react"
+import { Box } from "atomic-layout"
+import styled from "styled-components"
+
+const StyledBox = styled.div`
+  box-sizing: border-box;
+`
+
+export const Grid = ({ children, fluid }) => {
+  const maxWidths = fluid
+    ? { maxWidth: "1400px" }
+    : { maxWidthSm: "540px", maxWidthMd: "1200px" }
+
+  return (
+    <Box
+      as={StyledBox}
+      width="100%"
```

Comment:
It's the opposite: grids like this have `maxWidth="100%"` (which reads _never wider than the parent_) and different `width` based on the viewport size;

```diff
  const maxWidths = fluid
-    ? { maxWidth: "1400px" }
+    ? { width: "1400px" }
-    : { widthSm: "540px", widthMd: "1200px" }
+    : { widthSm: "540px", widthMd: "1200px" }
```
> `maxWidth` -> `width`

```diff
<Box
  as={StyledBox}
-  width="100%"
+  maxWidth="100%"
```

Reply:
I didn't know it. Thank you!